src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ * Sorts entries in the address book according to first name.
+ */
+
+public class SortCommand extends Command{
```

Comment:
coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)